main/src/main/java/org/apache/james/jdkim/DKIMVerifier.java

```diff
@@ -422,11 +422,22 @@ public List<Result> getResults() {
     }
 
     /**
-     * Returns true when all signature verification are successful. A message without dkim-signature is considered a success.
+     * Returns true when ANY signature verification is successful. A message without dkim-signature is considered a success.
+     * Must be called after verify method
      *
      * @return true when success
      */
     public boolean isSuccess() {
+        return result.stream().anyMatch(Result::isSuccess) || result.isEmpty();
+    }
```

Comment:
This modifies the result of the method.

Maybe we could keep isSuccees same as before (isAllSuccess) and have a hasAnyValidSignature method?

This avoids a breaking change.

Reply:
This function was added days ago, I tought it is not a breaking change since it was not released yet.

I will add another method with proper documentation then. I will keep the "isSuccess" returning true when all signatures match, and add the hasAnyValidSignature. I think there's no need to deprecate.

Reply:
Fair ^^